### ISSUE TITLE
Adding more weather conditions to the LGSVL simulator

### ISSUE THREAD

**Issue by kyjy1997** [lgsvl/simulator#1148]
Hi is there a possibility of adding more weather conditions such as snow just like fog, wetness etc and adjust it accordingly in the simulator gui

**Comment by EricBoiseLGSVL** [comment]
Yes you can add more weather features but adding to WebUI is difficult.  You can edit EnvironmentalEffectsManager.cs and ShaderGraphs/EnvironmentSimulation shader for the effects.  I'd add the control through the python API instead of the WebUI.
